docs/codegen-options.md

```diff
@@ -6,7 +6,7 @@
 | `graphqlSchemas`                                      | *See<br>[graphqlSchemas](#option-graphqlschemas)*                     | All<br>`.graphqls`/`.graphql`<br>files in<br>resources | Block to define the input GraphQL schemas, when exact paths are too cumbersome. See table below for a list of options. *See [graphqlSchemas](#option-graphqlschemas)* |
 | `graphqlQueryIntrospectionResu`<br>`ltPath`           | String                                                                | None                                                   | Path to GraphQL Introspection Query result in json format (with root object `__schema` or `data.__schema`). Sample: [sample-introspection-query-result.json](../src/test/resources/introspection-result/sample-introspection-query-result.json)|
 | `outputDir`                                           | String                                                                | None                                                   | The output target directory into which code will be generated. |
-| `jsonConfigurationFile`                               | String                                                                | Empty                                                  | Path to an external mapping configuration. |
+| `configurationFiles`                                   | List(String)                                                          | Empty                                                  | Path to an external mapping configuration. It support either JSON or HOCON. The same key is used in order, so the default configuration should be placed at the end.|
```

Comment:
Paths to the files with mapping configurations. Supported formats. JSON, HOCON. Order of specified configuration files matters, so the default configuration should be placed at the end.